.editorconfig

```diff
@@ -1,4 +1,16 @@
-# 2 space indentation
+# Top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file, utf-8 charset
+[*]
+end_of_line = lf
+trim_trailing_whitespace = true
+charset = utf-8
+
 [*.py]
 indent_style = space
-indent_size = 2
```

Comment:
why was this removed?

Reply:
1) Because it is now enforced by pylint.

2) Editorconfig does not handle this properly as there are cases when this strict rule needs to be broken: when function argument list is split into two lines, the second line can start on both even and odd column, depending on the function name.